Forms/index.html

```diff
@@ -9,82 +9,96 @@
 </head>
 
 <body>
-    <img src="res/left-img.png" class="img1">
-    <h1>
-        <font size="15" face="segoe print">Алко.net</font>
-    </h1>
+    <div class="head-container">
+        <h1>
+            <font size="15" face="segoe print">Алко.net</font>
+        </h1>
+        <img src="res/left-img.png" class="img1">
+    </div>
 
     <div class="container">
-        <form action="#" method="get" enctype="multipart/form-data" id="form">
-            <div class="first_collum">
+        <div class="first_column">
+            <div>Текст </div>
+        </div>
+        <form id="form">
+            <div class="second_column">
                 <p>
                 <h2>Мы проводим анкетирование, пожалуйста пройдите опрос</h2>
                 </p>
                 <div class="text_form">
```

Comment:
It's semantically incorrect to name divs like that. There are special tags for it - form. If this div is a part of a form, then it's classname should be something like "name_input", i.e. something, describing what's happening inside that div. But surely not the "form"